database/index.js

```diff
@@ -0,0 +1,86 @@
+const mysql = require('mysql');
+const mysqlConfig = require('./config.js');
+const { Sequelize } = require('sequelize');
+const { user } = require('./config.js');
+const sequelize = new Sequelize('humblebundle', 'root', '', { host: 'localhost', dialect: 'mysql'});
+const Seed = require('./seed.js');
+
+const connection = mysql.createConnection(mysqlConfig);
+
+var Bundles = sequelize.define('bundles', {
+  bundleId: Sequelize.INTEGER,
+  tierId: Sequelize.INTEGER,
+  cost: Sequelize.INTEGER,
+  tierOrder: Sequelize.INTEGER
+});
+
+var Tiers = sequelize.define('tiers', {
+  tierId: Sequelize.INTEGER,
+  itemId: Sequelize.INTEGER,
+  itemOrder: Sequelize.INTEGER
+});
+
+
+// Bundles.drop()
+// .then(() => {
+//   Bundles.sync({ force: true })
+//   .then(() => {
+//     data = Seed.seedBundles();
+//     Bundles.bulkCreate(data, { validate: true })
+//     .then(() => {
+//       console.log('bundles created');
+//     })
+//     .catch(err => {
+//       console.log('failed to create bundles', err);
+//     })
+//     .finally(() => {
+//       sequelize.close();
+//     })
+//   })
+// });
+
+// Tiers.drop()
+// .then(() => {
+//   Tiers.sync({ force: true })
+//   .then(() => {
+//     data = Seed.seedTiers();
+//     Tiers.bulkCreate(data, { validate: true})
+//     .then(() => {
+//       console.log('tiers created');
+//     })
+//     .catch(err => {
+//       console.log('failed to create tiers', err);
+//     })
+//     .finally(() => {
+//       sequelize.close();
+//     })
+//   })
+// });
+
+const tiersRequest = (data) => {
+  let bundleId = data.bundleId;
+  return Bundles.findAll({
+    where: {
+      bundleId: bundleId
+    }
+  })
+  .then(data => {
+    return data;
+  })
+};
+
+const itemsRequest = (data) => {
+  let tierId = data.tierId;
+  return Tiers.findAll({
+    where: {
+      tierId: tierId
+    }
+  })
+  .then(data => {
+    return data;
+  })
+};
```

Comment:
Are you choosing to swallow any errors that occur here (in both of these functions)? If so, I would write some sort of code to indicate the choice. Otherwise, a `.catch` could be an option.